src/unix/mod.rs

```diff
@@ -100,6 +106,47 @@ pub const DT_REG: u8 = 8;
 pub const DT_LNK: u8 = 10;
 pub const DT_SOCK: u8 = 12;
 
+pub const FD_CLOEXEC: ::c_int = 0x1;
+
+pub static CLONE_VM: ::c_uint = 0;
+pub static CLONE_FS: ::c_uint = 0;
+pub static CLONE_FILES: ::c_uint = 0;
+pub static CLONE_SIGHAND: ::c_uint = 0;
+pub static CLONE_PTRACE: ::c_uint = 0;
+pub static CLONE_VFORK: ::c_uint = 0;
+pub static CLONE_PARENT: ::c_uint = 0;
+pub static CLONE_THREAD: ::c_uint = 0;
+pub static CLONE_NEWNS: ::c_uint = 0;
+pub static CLONE_SYSVSEM: ::c_uint = 0;
+pub static CLONE_SETTLS: ::c_uint = 0;
+pub static CLONE_PARENT_SETTID: ::c_uint = 0;
+pub static CLONE_CHILD_CLEARTID: ::c_uint = 0;
+pub static CLONE_DETACHED: ::c_uint = 0;
+pub static CLONE_UNTRACED: ::c_uint = 0;
+pub static CLONE_CHILD_SETTID: ::c_uint = 0;
+pub static CLONE_NEWUTS: ::c_uint = 0;
+pub static CLONE_NEWIPC: ::c_uint = 0;
+pub static CLONE_NEWUSER: ::c_uint = 0;
+pub static CLONE_NEWPID: ::c_uint = 0;
+pub static CLONE_NEWNET: ::c_uint = 0;
+pub static CLONE_IO: ::c_uint = 0;
```

Comment:
d'oh, looks like the usage of `pub static` instead of `pub const` caused these to all subvert CI, causing mayhem!